projects/tod_simulator/README.md

```diff
@@ -71,9 +87,63 @@ Given a conversation generated from `tod_world_script`, outputs statistics about
 
 **tod\_distributed\_uber\_multiwoz\_script.py**
-Version of `tod_distributed_uber_script.py` but with MultiWoz v2.2 as the primary task rather than Google SGD. (Included for completeness.)
+Version of `tod_distributed_uber_script.py` but with MultiWoz v2.2 as the primary task rather than Google SGD Out-of-Domain. (Included for completeness.)
 
 **tod\_distributed\_uber\_script.py**
-Multi-step train, evaluation, and data generation script used in Simulations paper. Uses Google SGD as primary dataset; note "STANDALONE\_API\_FILE\_PATH" that needs to be set in file. Makes use of `do_get_passing_only_on_dir.py` and `get_al_samples_for_gsgd.py`; use `get_passing_only.py` and `get_api_data.py` after the fact for analysis. 
+Multi-step train, evaluation, and data generation script used in Simulations paper. Uses Google SGD Out-of-Domain as primary dataset; note "STANDALONE\_API\_FILE\_PATH" that needs to be set in file. Makes use of `do_get_passing_only_on_dir.py` and `get_al_samples_for_gsgd.py`; use `get_passing_only.py` and `get_api_data.py` after the fact for analysis. 
 
 Note that this script is intended to be run in a SLURM environment matching that of the Simulations paper authors. It is unknown how the script performs in other settings but is included as a reference.
+
+## Tasks used in the paper 
+
+See the appendix of [the paper](https://arxiv.org/abs/2110.06905) (or the description of the task in ParlAI Task List) for explanations of these datasets. Below, we include the dataset name, the command to run the `SystemTeacher` relevant for each of the datasets, and any other notable details. Other agents and teachers for the dataset are specified in the relevant task `agent.py` files. 
+
+### Pretraining Tasks
+
+* Google SGD In-Domain 
+    * `parlai dd -t google_sgd_simulation_splits:InDomainSystemTeacher`
+* MetalWoz
+    * `parlai dd -t metalwoz:SystemTeacher`
+* MSR_E2E
+    * `parlai dd -t msr_e2e:SystemTeacher`
+    * Note that due to the lack of annotations in this dataset, this System Teacher *only* includes utterance turns
+* Multidogo
+    * `parlai dd -t multidogo:SystemTeacher`
+* MultiWoz 
+    * We use a fb-internal pre-processing of MultiWoz, based on MultiWoz v2.1 and do not open source it at this time.
+* Taskmaster
+    * `parlai dd -t taskmaster:SystemTeacher`
+* Taskmaster2
+    * `parlai dd -t taskmaster2:SystemTeacher`
+* Taskmaster3 (TicketTalk)
+    * `parlai dd -t taskmaster3:SystemTeacher`
+
+### Experimentation Tasks
+
+* Google SGD Out-of-Domain
+   * `parlai dd -t google_sgd_simulation_splits:OutDomainSystemTeacher`
+* MultiWoz (not currently included in paper) 
+   * `parlai dd -t multiwoz_v22:SystemTeacher`
+   * This is a preprocessing of the dataset based on MultiWoz v2.2. Though utterances are the same as used for pre-training, API Call and API Response structures aer different. 
+
+See "scripts in project directory" for scripts associated with training, evaluation, and data generation. 
```

Comment:
Super minor - maybe clarify that this refers to another section in this README, right?